core/data/tests/test_generate_char/output.py

```diff
@@ -0,0 +1,12 @@
+"""
+ Generated by typeshare 1.12.0
+"""
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+class MyType(BaseModel):
+    field: str
```

Comment:
Consider `constr` to constrain the length here, since it's known to be a `char` https://docs.pydantic.dev/1.10/usage/types/#arguments-to-constr

Reply:
Will be using Annotated with Field instead.

Reply:
Marking as won-fix to prevent scope-creeping.